app/components/panel/body.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="vlc-panel__body" ...attributes>
```

Comment:
This is the first in a series of pull requests, we discussed component hierarchy with @madnificent on Rocket.chat and landed on this kind of logic. Not sure about the suddenness of this.

Reply:
If we don't put them straight there we risk having `<Overly::Long::Component::Names />`.

Reply:
The repo is already chaotic as it is. Please follow the current structure.

Reply:
What structure do you propose? Where should I put this component?

Reply:
The convention that was already in place. Every component has at least a folder in the components folder. We just want to prevent a bit of a soup here. Go for `app/Panel/...` perhaps?

Reply:
I changed it so there are no “stray” .hbs/.js files in the `components` root folder.

Reply:
That stray component was the only thing that worried me, so now I love you again @Wolfr 😘

Reply:
There was some confusion between "flat" and "nested" structure here ( <https://github.com/ember-codemods/ember-component-template-colocation-migrator> ). Currently stick with how it is. On the other hand it's right that "flat" is the new default. (what you would end up with when using `ember g ...` I believe ...)